.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Run './gradlew spotlessApply'
+2341806496ba2f23532a42775c4ef53b965e07c0
```

Comment:
**Important**: Don't squash or rebase when merging as this'll invalidate this rev.